lib/business/data/betalingsservice.yml

```diff
@@ -44,11 +44,23 @@ holidays:
   - January 1st, 2020
   - April 9th, 2020
   - April 10th, 2020
-  - April 12th, 2020
   - April 13th, 2020
   - May 8th, 2020
-  - May 31st, 2020
+  - May 21st, 2020
+  - May 22nd, 2020
   - June 1st, 2020
   - June 5th, 2020
+  - December 24th, 2020
   - December 25th, 2020
   - December 26th, 2020
+  - December 31st, 2020
+  - January 1st, 2021
+  - April 1st, 2021
+  - April 2nd, 2021
+  - April 5th, 2021
+  - April 30th, 2021
+  - May 13th, 2021
+  - May 14th, 2021
+  - May 24th, 2021
+  - December 24th, 2021
```

Comment:
And the 25th too I'm guessing?

Reply:
It's on a saturday next year but i'll add it anyway